sdk/servicebus/Azure.Messaging.ServiceBus/tests/Client/ServiceBusClientTests.cs

```diff
@@ -226,6 +226,7 @@ public void ConstructorValidatesConnectionStringForDuplicateAuthorization(string
             Assert.That(options, Is.Not.SameAs(defaultOptions), $"The { constructorDescription } constructor should not have the same options instance.");
```

Comment:
We should also consider bringing these over if we don't have equivalents.... these were added as part of the custom endpoint address work in Event Hubs:   
https://github.com/Azure/azure-sdk-for-net/pull/18014/files#diff-d317b71217720aff41196242e8f784de2e9cc76d78f0952785198e2a13ede7f0